src/includes/getting-started-primer/apple.mdx

```diff
@@ -0,0 +1,27 @@
+<Note>
+
+*Sentry's SDKs report an error automatically whenever a thrown error or exception goes uncaught in your application causing the application to crash. Sentry's iOS, tvOS, and watchOS SDK hook into all signal and exception handlers, except for MacOS.*
+
+*The SDK builds a crash report that persists to disk and tries to send the report right after the crash. Since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.*
+
+</Note>
+
+**Features:**
+
+- Events [enriched](/platforms/apple/enriching-events/context/) with device data
+- Offline caching when a device is offline; we send a report once we receive another event
```

Comment:
`Offline caching when a device is offline` seems repeating, what's about, when a device is running out of connectivity or something

Reply:
Completely agree. Thanks for a second set of eyes on this, @marandaneto